src/config.ts

```diff
@@ -15,6 +15,7 @@ export const DEFAULT_MAX_LIMIT = 500;
 export const DEFAULT_VERBOSE = false;
 export const APP_NAME = pkg.name;
 export const DEFAULT_SORT_BY = "DESC";
+export const DEFAULT_AGGREGATE_FUNCTION = "count";
```

Comment:
this should be `sum`
https://clickhouse.com/docs/en/sql-reference/aggregate-functions/reference/sum

`count` only counts the number of rows
https://clickhouse.com/docs/en/sql-reference/aggregate-functions/reference/count